tools/output-number-of-buffer-segments.rb

```diff
@@ -0,0 +1,31 @@
+#!/bin/ruby
+
+require "open3"
+require "json"
+
+if ARGV.size() == 0
+  puts "Usage: $0 DATABASE_PATH"
+  puts " e.g.: $0 ~/database/db"
+  exit
+end
+
+DATABASE_PATH = ARGV[0]
+stdout, stderr, status = Open3.capture3("groonga",
+                                        DATABASE_PATH,
+                                        "dump",
+                                        "--dump_plugins","no",
+                                        "--dump_schema", "no",
+                                        "--dump_records", "no",
+                                        "--dump_configs", "no")
```

Comment:
How about check the status and output `stdout` and `stderr` if an error occurred?

Reply:
Thank you for your comment.
I see. I've added them at https://github.com/groonga/groonga/pull/1192/commits/e2273898490c01b39621a408e7c9f18edae9cbb3.